Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM rocker/tidyverse:3.6.3
 
+RUN apt-get update && apt-get install -y \
+    libudunits2-dev \ 
+    libgdal-dev \
+    libgeos-dev \
```

Comment:
Technically not necessary to include `libgeos-dev` and `libproj-dev` explicitly since they are installed alongside `libgdal-dev`, but doesn't hurt to be explicit following the `sf` package documentation.